server/src/main/java/org/elasticsearch/rest/action/admin/cluster/RestClusterRerouteAction.java

```diff
@@ -70,6 +77,14 @@ public boolean allowSystemIndexAccessByDefault() {
 
     @Override
     public RestChannelConsumer prepareRequest(final RestRequest request, final NodeClient client) throws IOException {
+        if (request.hasParam("metric") == false || Objects.equals(request.param("metric"), "none") == false) {
```

Comment:
Oh also could we somehow mention a 7.x version constant (e.g. `Version#V_7_17_0`) so that we get a compiler error in 9.x to trigger the followup actions there?